REMAINING.PUBLIC.md

```diff
@@ -0,0 +1,70 @@
+### Remaining public aliases
+
+annotation_reader
+cache.app_clearer
+$commandId
+debug.templating.engine.php
+profiler.request_matcher
+router
+security.access.decision_manager
+security.acl.cache
+security.acl.dbal.connection
```

Comment:
this should not need to be public (if you need the DBAL connection yourselves dynamically, you should get it using the public ids given in DoctrineBundle)